quickinfo/quickinfo-war/src/main/java/org/silverpeas/components/quickinfo/web/TickerResource.java

```diff
@@ -0,0 +1,41 @@
+package org.silverpeas.components.quickinfo.web;
+
+import org.silverpeas.components.quickinfo.model.News;
+import org.silverpeas.core.annotation.RequestScoped;
+import org.silverpeas.core.annotation.Service;
+import org.silverpeas.core.webapi.base.UserPrivilegeValidation;
+import org.silverpeas.core.webapi.base.annotation.Authenticated;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+@Service
+@RequestScoped
+@Path("news/ticker")
+@Authenticated
+public class TickerResource extends AbstractNewsResource {
+
+  @Override
+  public String getComponentId() {
+    return null;
+  }
+
+  @Override
+  public void validateUserAuthentication(final UserPrivilegeValidation validation)
+      throws WebApplicationException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'javax.ws.rs.WebApplicationException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)